xarray/core/computation.py

```diff
@@ -1375,6 +1388,8 @@ def corr(da_a: T_DataArray, da_b: T_DataArray, dim: Dims = None) -> T_DataArray:
         Array to compute.
     dim : str, iterable of hashable, "..." or None, optional
         The dimension along which the correlation will be computed
+    weights : DataArray, default: None
+        Array of weights.
```

Comment:
I would like to see an intuitive example using this parameter. 

Could you add one for `cov` and `corr`, there's an Examples section further down.

Reply:
I have added examples in the last commit.